app/panel/components/Settings/SettingsMenu.jsx

```diff
@@ -14,6 +14,10 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import ClassNames from 'classnames';
+import globals from '../../../../src/classes/Globals';
+
+const IS_CLIQZ = (globals.BROWSER_INFO.name === 'cliqz');
```

Comment:
Why not use `globals.IS_CLIQZ`?

Reply:
good call, that's in a few other places too.  let me remove.